MAUI/PDF-Viewer/Built-In-Toolbar.md

```diff
@@ -0,0 +1,741 @@
+---
+layout: post
+title: Working with the Built-In Toolbar in .NET MAUI Pdf Viewer (SfPdfViewer) | Syncfusion
+description: Learn here all about the built-in toolbar in .NET MAUI PDF Viewer (SfPdfViewer) control, its functionalites and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+keywords : .net maui pdf viewer, .net maui view pdf, pdf viewer in .net maui, .net maui open pdf, maui pdf viewer, maui pdf view
+---
+
+# Working with the Built-In Toolbar in .NET MAUI Pdf Viewer (SfPdfViewer)
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) has a built-in toolbar, which allows you to perform operations such as adding, modifying annotations, searching text, etc. Additionally, you have the option to show or hide the default built-in toolbar and its sub-toolbars and toolbar items, such as buttons, and you can customize your own toolbar by disabling the built-in toolbar and implementing one at the application and hooking the custom toolbar UI to the PDF viewer APIs. You can see the custom toolbar example [here](https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/tree/master/Custom%20Toolbar).
```

Comment:
<img width="225" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/97388e0d-e1e2-418f-bb0a-09cbc89686b4">

Use different word for this